### PR TITLE
Add and use self-hosted fonts

### LINKED ISSUE #120: Use self hosted Google Fonts
Use self hosted Google Fonts instead of loaded over Google API.

Improve compatibility to [GDPR](https://cookie-script.com/blog/google-fonts-and-gdpr).

### PR DESCRIPTION
**Pull request by SSirjes**
Add and use self hosted fonts to improve GDPR compliance.
Closes #120

### CODE REVIEW

File: sources/stylesheets/_fonts.scss [hunk 1]
```diff
@@ -0,0 +1,530 @@
+@font-face {
```
Comment (pxska) [inline sources/stylesheets/_fonts.scss:1]: Indentation in this file should be fixed: everything below this line should be indented two spaces to the left.

This file should also have an empty line at the end of the file (so GitHub wouldn't be mad at us).

**Review by pxska — APPROVED** [review]
One small comment, otherwise looks good!

File: sources/stylesheets/_fonts.scss [hunk 2]
```diff
@@ -0,0 +1,530 @@
...
+  src: url('../assets/roboto-v30-vietnamese_latin-ext_greek-ext_greek_cyrillic-ext_cyrillic-300.woff') format('woff');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Roboto';
+  src: url('../assets/roboto-v30-vietnamese_latin-ext_greek-ext_greek_cyrillic-ext_cyrillic-300italic.woff') format('woff');
+  font-weight: 300;
+  font-style: italic;
+}
```
Comment (PeeterPaal) [inline sources/stylesheets/_fonts.scss:375]: These are actually not necessary, as this design only uses 400 (`normal`) and 700 (`bold`) font weights. But nothing will break by leaving them in :)

File: sources/stylesheets/_fonts.scss [hunk 3]
```diff
@@ -0,0 +1,530 @@
...
+  src: url('../assets/roboto-v30-vietnamese_latin-ext_greek-ext_greek_cyrillic-ext_cyrillic-500.woff') format('woff');
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: 'Roboto';
+  src: url('../assets/roboto-v30-vietnamese_latin-ext_greek-ext_greek_cyrillic-ext_cyrillic-500italic.woff') format('woff');
+  font-weight: 500;
+  font-style: italic;
+}
```
Comment (PeeterPaal) [inline sources/stylesheets/_fonts.scss:401]: Previous comment also applies here.

**Review by PeeterPaal — APPROVED** [review]
Just a few remarks, but overall looks good!

File: stylesheets/main.css [hunk 4]
```diff
@@ -369,6 +369,537 @@
...
+@font-face {
+  font-family: 'Raleway';
+  src: url("../assets/raleway-v28-vietnamese_latin-ext_cyrillic-ext_cyrillic-700italic.woff") format("woff");
+  font-weight: bold;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Roboto';
+  src: url("../assets/roboto-v30-vietnamese_latin-ext_greek-ext_greek_cyrillic-ext_cyrillic-300.woff") format("woff");
```
Comment (tanelj) [inline stylesheets/main.css:737]: Roboto 300 and 500 rules are removed form source (.scss) files but are still present in style.css.

Please run `grunt`.

**Review by tanelj — CHANGES_REQUESTED** [review]
Nice, but style.css is not updated.

**Review by tanelj — APPROVED** [review]
Nice!

**Review by PeeterPaal — APPROVED** [review]

**Pull request merged**